src/data/context.ts

```diff
@@ -30,3 +31,5 @@ export const fullEntitiesContext =
 export const floorsContext = createContext<HomeAssistant["floors"]>("floors");
 
 export const labelsContext = createContext<LabelRegistryEntry[]>("labels");
+
+export const configEntries = createContext<ConfigEntry[]>("configEntries");
```

Comment:
The others are named `xxxContext`
```suggestion
export const configEntriesContext = createContext<ConfigEntry[]>("configEntries");
```